root/static/scripts/common.js

```diff
@@ -1,3 +1,8 @@
+// Needed by root/release/cover_art_uploader.tt, which uses the
+// css_manifest TT macro that requires common.less to exist in
+// rev-manifest.json.
+require('../styles/common.less');
```

Comment:
The `css_manifest` TT macro uses `Data::FileCache` under the hood, which tries to look things up in rev-manifest.json. This is to make sure it gets added there for the cover art uploader.

There's a `require('../../static/styles/common.less')` already present in root/layout/components/Head.js, but that does *not* add it to the manifest, because that file is only included in the server build, and the manifest is generated from the client build.

Reply:
So note that for React-rendered pages, we don't need CSS or image files in rev-manifest.json at all - the URIs for those resources are got via `file-loader`, as in root/layout/components/Head.js. This is just a hack for an old TT page which expects CSS in the manifest.